### DISCUSSION TITLE
6.11 二分图基础

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#180] [Comments]
# 6.11 二分图基础

二分图 本章内容 二分图基础知识 二分图基础题目 二分图最大匹配知识 匈牙利算法 Hopcroft-Karp 算法 二分图最大匹配题目

[https://algo.itcharge.cn/06_graph/06_11_graph_bipartite_basic/](https://algo.itcharge.cn/06_graph/06_11_graph_bipartite_basic/)

**Comment by T1anq1** [comment]
二分图整个条目内容似乎都消失了？

**Reply by itcharge** [reply]
已经更新～
